docs/pages/router/advanced/shared-routes.mdx

```diff
@@ -43,3 +42,27 @@ export default function DynamicLayout({ segment }) {
   return <Stack />;
 }
 ```
+
+To enable the **array syntax**, specify the [`initialRouteName`](/router/advanced/router-settings/#initialroutename) for each group using `unstable_settings` object in the dynamic layout:
+
+{/* prettier-ignore */}
+```tsx app/(home,search)/_layout.tsx
+export const unstable_settings = {
+  intialRouteName: 'home',
+  search: {
+    initialRouteName: 'search',
+  },
+};
+
+export default function DynamicLayout({ segment }) {
+  /* @hide  ... */ /* @end */
+}
+```
+
+In the above example, the `home` route is the default route for the `home` group and the app. The `search` route is the default route for the `search` group.
+
+## Key points
+
+- You can only provide groups for the current navigator.
```

Comment:
@marklawlor, can you elaborate more on this point? I'm not sure what "current navigator" means here.